hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
That part is not solved yet, which is extremely unfortunate. "rebuild &
push periodically" is the best answer I have right now. We need to run a
private registry in e2e clusters so there's a place to push & pull these
random containers to/from.

On Mon, Oct 5, 2015 at 1:28 PM, Stephen Milner <notifications@github.com>
wrote:

> In hack/build-cross.sh
> <https://github.com/kubernetes/kubernetes/pull/11694#discussion_r41193007>
> :
>
> > @@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
> >  kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
> >
> >  KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
> > +KUBE_STATIC_OVERRIDES=("kubectl")
>
> Looks like /examples/kubectl-container/Makefile sets KUBE_STATIC_OVERRIDES
> to "kubectl".
>
> @lavalamp <https://github.com/lavalamp> how would you recommend executing
> and referencing it in jenkins e2e? As part of the e2e test itself before
> uploading into netexec?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/11694/files#r41193007>.
>